### PR TITLE
chore: Add repo files

### PR DESCRIPTION
**Pull request by MeikelLP**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿# Security
+
+**Do not report vulnerabilities through public GitHub issues, discussions, pull requests, or any other public form of communication.**
+
+Please directly email <quantum-core-x-security@meikel.net> to report vulnerabilities.  Depending on the severity of the vulnerability, and to our sole discretion, we may choose to award a bug bounty for the vulnerability disclosure.
+
+To send us a PGP encrypted email, download our key at <https://keys.openpgp.org/search?q=quantum-core-x-security%40meikel.net>
```
Comment (FatalMerlin) [inline SECURITY.md:7]: Should we add the key Fingerprint?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,129 +1,139 @@
...
-  and learning from the experience
-* Focusing on what is best not just for us as individuals, but for the
-  overall community
+## Code of Conduct
 
-Examples of unacceptable behavior include:
+This project and everyone participating in it is governed by the
+[Quantum Core X Code of Conduct](https://github.com/MeikelLP/quantum-core-x/blob/master/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior
+to .
```
Comment (FatalMerlin) [inline CONTRIBUTING.md:32]: To whom? The suspense is killing me!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,129 +1,139 @@
...
 
-## Scope
+If you then still feel the need to ask a question and need clarification, we recommend the following:
 
-This Code of Conduct applies within all community spaces, and also applies when
-an individual is officially representing the community in public spaces.
-Examples of representing our community include using an official e-mail address,
-posting via an official social media account, or acting as an appointed
-representative at an online or offline event.
+- Open an [Issue](https://github.com/MeikelLP/quantum-core-x/issues/new).
```
Comment (FatalMerlin) [inline CONTRIBUTING.md:43]: Follow-up: Is there a good issue template generator?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,129 +1,139 @@
...
 
-For answers to common questions about this code of conduct, see the FAQ at
-https://www.contributor-covenant.org/faq. Translations are available at
-https://www.contributor-covenant.org/translations.
+Refer to [C# identifier names](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names) as the default style guide. It is the default in Visual Studio and Rider.
+Any override will be enforced by tools (.editorconfig or similar) from us - so don't worry.
+
+### Commit Messages
+
+We are using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
```
Comment (FatalMerlin) [inline CONTRIBUTING.md:131]: Are we enforcing this? Maybe add an explicit line if so, e.g. `Pull Requests and commits that do not follow this standard will not be reviewed and instead immediately rejected.`

**Review by FatalMerlin — CHANGES_REQUESTED** [review]
I have some minor improvement suggestions and questions, see my notes, otherwise looks good!

File: CONTRIBUTING.md (hunk 4 above)
Comment (MeikelLP) [reply CONTRIBUTING.md:131]: Sounds too harsh for me. Contributions should be welcomed not restricted. We can always squash and adjust commit messages 👍

Comment (FatalMerlin) [reply CONTRIBUTING.md:131]: Oh, yeah, good point!

**Review by FatalMerlin — APPROVED** [review]

**Pull request merged**